.gitignore

```diff
@@ -0,0 +1,101 @@
+### Go template
```

Comment:
Please remove this file. We can discuss it an issue and the maybe add it in a separate PR.